static/js/i18n/strings/en/stats_var_titles.json

```diff
@@ -87,6 +87,34 @@
     "defaultMessage": "Number of Criminal Activities (Violent Crimes)",
     "description": "Title for a page about a statistical variable. Variable describes total number of violent crimes in a place."
   },
+  "Count_CriminalIncidents_IsHateCrime": {
+    "defaultMessage": "Number of Criminal Hate Crime Incidents",
+    "description": "Title for a page about a statistical variable. Variable describes total number of hate crime incidents in a place."
+  },
+  "Count_CriminalIncidents_BiasMotivationGender_IsHateCrime": {
+    "defaultMessage": "Number of Criminal Hate Crime Incidents",
```

Comment:
Template for this and the following:
"Number of Criminal Incidents (Hate Crimes Motivated by Gender)"

Reply:
fixed.
I totally missed changing the defaultMessage string. 

Thanks for the review and support @beets :)